.github/actions/configure-aws-credentials/action.yml

```diff
@@ -54,7 +54,7 @@ runs:
 
     - name: Configure AWS credentials for dev account
       uses: aws-actions/configure-aws-credentials@v4
-      if: ${{ inputs.account-name == 'dev' || inputs.account-name == 'dpd' }}
+      if: ${{ contains(fromJSON('["dev", "dpd", "auth-dev"]'), inputs.account-name) }}
```

Comment:
`contains()` expects a string not an array, hence the need for this janky parsing of an array into a string. I could maybe have done something like:
```
contains("dev,dpd,auth-dev", inputs.account-name)
```
But that felt almost worse